go-controller/pkg/metrics/ovn_northd.go

```diff
@@ -35,6 +40,22 @@ func getOvnNorthdVersionInfo() {
 	}
 }
 
+func getOvnNorthdConnectionStatusInfo(command string) float64 {
+	stdout, stderr, err := util.RunOVNNorthAppCtl(command)
```

Comment:
how often will this be run? is there any performance impact?

@igsilya @dceara fyi

Reply:
@trozet There should be no significant performance impact.  ovn-northd's handling of appctl commands doesn't trigger reconciliation of database contents since incremental processing has been added to ovn-northd.  The only exception is the `inc-engine/recompute` command but that's not the case here.  Didn't test and this is not a proper review but LGTM.

Reply:
I tried running it every .5 seconds. No impact to CPU. Tried running it every .1 seconds, also no impact. Running every .01 seconds I saw a CPU increase of about 1.6%. I doubt we are polling anywhere close to that. I think this is fine to merge.